ntc_templates/templates/cisco_ios_show_lldp_neighbors_detail.textfsm

```diff
@@ -1,5 +1,5 @@
 Value LOCAL_INTERFACE (\S+)
-Value CHASSIS_ID (\S+)
+Value CHASSIS_ID ([^:]*)
```

Comment:
since matching anything not colon(:) maybe also exclude blank space ?
Should the match be nothing (*) or at least something (+) ?
e.g. ([^:\s]+)

Reply:
@diepes Thanks for the feedback. I've pushed a commit now using `+` instead of `*`.

My example in the PR wasn't very clear for this case.
The output I get from a device is:
`Chassis id: S7-1200                   6DS7 212-1BH40-0XB0  S V-LCCC5820        10 V  4  3  1`

Do you think its valid to capture the output chassis-id as the following (including spaces)?:
`S7-1200                   6DS7 212-1BH40-0XB0  S V-LCCC5820        10 V  4  3  1`